utils/textmate/Syntaxes/carbon.tmLanguage.json

```
@@ -0,0 +1,221 @@
+{
+  "$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+  "name": "carbon",
+  "foldingStartMarker": "\\{\\s*$",
+  "foldingStopMarker": "^\\s*\\}",
+  "fileTypes": [
+    "carbon"
+  ],
+  "patterns": [
+    {
+      "include": "#comments"
+    },
+    {
+      "include": "#operators"
+    },
+    {
+      "name": "string.quoted.triple.carbon",
+      "begin": "'''([^\\s'#]*\\n)?",
+      "end": "'''",
+      "beginCaptures": {
+        "1": {
+          "name": "constant.character.escape.carbon"
+        }
+      },
+      "patterns": [
+        {
+          "include": "#string_escapes"
+        }
+      ]
+    },
+    {
+      "name": "string.quoted.double",
+      "begin": "\"([^\\s'#]*\\n)?",
+      "end": "\"",
+      "beginCaptures": {
+        "1": {
+          "name": "constant.character.escape.carbon"
+        }
+      },
+      "patterns": [
+        {
+          "include": "#string_escapes"
+        }
+      ]
+    },
+    {
+      "include": "#strings"
+    },
+    {
+      "include": "#special-keywords"
+    },
+    {
+      "include": "#numbers"
+    },
+    {
+      "include": "#reserved-words"
+    },
+    {
+      "include": "#operator-dedicated-keywords-statements"
+    },
+    {
+      "include": "#ctrl-statements"
+    },
+    {
+      "include": "#true-false"
+    },
+    {
+      "include": "#functions"
+    },
+    {
+      "include": "#customs"
+    }
+  ],
+  "repository": {
+    "operators": {
+      "patterns": [
+        {
+          "name": "keyword.operator.new.carbon",
+          "match": "\\b(\\+|-|\\*|/|!)\\b"
+        }
+      ]
+    },
+    "comments": {
+      "patterns": [
+        {
+          "name": "comment.line.double-slash.carbon",
+          "begin": "//",
+          "end": "\n"
+        }
+      ]
+    },
+    "strings": {
+      "patterns": [
+        {
+          "name": "string.quoted.single.carbon",
+          "begin": "'",
+          "end": "'"
+        },
+        {
+          "name": "string.quoted.double.carbon",
+          "begin": "\"",
+          "end": "\""
+        },
+        {
+          "name": "string.quoted.triple.carbon",
+          "begin": "'''",
+          "end": "'''"
+        }
+      ]
+    },
+    "string_escapes": {
+      "patterns": [
+        {
+          "name": "constant.character.escape.carbon",
+          "match": "\\\\([tnr'\"0\\0]|x[0-9A-F]{2}|u\\{[0-9A-F]{4,}\\})"
+        }
+      ]
+    },
+    "special-keywords": {
+      "patterns": [
+        {
+          "name": "keyword.control.carbon",
+          "match": "\\b(break|case|continue|default|else|if|for|match|return|returned|then|while)\\b"
+        }
+      ]
+    },
+    "operator-dedicated-keywords-statements": {
+      "patterns": [
+        {
+          "name": "keyword.operator.new.carbon",
```

The things being matched here aren't operators. Maybe
```suggestion
          "name": "keyword.other.carbon",
```